engine/apps/mattermost/models/user.py

```diff
@@ -0,0 +1,12 @@
+from django.db import models
+
+
+class MattermostUser(models.Model):
+    user = models.OneToOneField("user_management.User", on_delete=models.CASCADE, related_name="mattermost_connection")
+    mattermost_user_id = models.CharField(max_length=100)
+    username = models.CharField(max_length=100)
+    nickname = models.CharField(max_length=100, null=True, blank=True, default=None)
+    created_at = models.DateTimeField(auto_now_add=True)
+
+    class Meta:
+        unique_together = ("user", "mattermost_user_id")
```

Comment:
Hm... Should `mattermost_user_id` be unique/index and then (since we have a `OneToOneField` relation with user) we may not need this? (so a Mattermost user can be only linked once, and to one user only)

Reply:
Now when i give a 2nd thought about it, for our current use cases I feel we will fetch the mattermost user from the oncall user object only so we might not need it right now. Removing this index now.